main.rb

```diff
@@ -0,0 +1,102 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'classroom'
+
+def list_people
+  Student.all.each do |person|
+    puts "[student] Name:#{person.name} ID:#{person.id} Age: #{person.age}"
+  end
+  Teacher.all.each do |person|
+    puts "[teacher] Name:#{person.name} ID:#{person.id} Age: #{person.age}"
+  end
+end
+
+def list_books
+  Book.all.each_with_index do |book, index|
+    puts "#{index}) Title:#{book.title}, Author:#{book.author}"
+  end
+end
+
+def create_person
+  default_class = Classroom.new('science')
+
+  print 'Do you want to create a student (1) or a teacher (2)?[input the number]: '
+  role = gets.chomp.to_i
+  print 'Age: '
+  age = gets.chomp.to_i
+  print 'Name: '
+  name = gets.chomp
+
+  if role == 1
+    print 'Has parent permission? [y/n]: '
+    permission = gets.chomp == 'y'
+    Student.new(age, default_class, name, permission)
+  else
+    print 'Specialization: '
+    specialization = gets.chomp
+    Teacher.new(age, specialization, name)
+  end
+
+  puts 'Person created succesfully'
+end
+
+def create_book
+  print 'Title: '
+  title = gets.chomp
+  print 'Author: '
+  author = gets.chomp
+  Book.new(title, author)
+
+  puts 'Person created succesfully'
+end
+
+def create_rental
```

Comment:
- [ ] **The app doesn't display the list of people to select from when creating a rental. Please fix this.**

When the app prompts the user to select a person while creating a rental, it should display the list of people to choose from. However, your app doesn't display a list of people. Please fix this issue. 

![listofpersonandrentaldateerror](https://user-images.githubusercontent.com/59168430/134193990-81239abc-aad0-4a65-9260-07a72127ba74.gif)

Check your `create_rental` method. Are you ever creating an instance of `people`?